### PR TITLE
Possible fix to #329

### PR DESCRIPTION
**Pull request by SimplyBallistic**
In reference to #329 and [post found here.](https://www.spigotmc.org/threads/need-bug-squasher.248782/#post-2483774).
My spigot account is xBallisticBlazex. You can report if this fixed your issue there. 
I also performed some code cleanup.

### CODE REVIEW

**Comment by SimplyBallistic** [comment]
Currently working on something I found from the post you made on spigot

**Comment by FlyingPikachu** [comment]
@SimplyBallistic Okay, I'll disregard this commit. Thanks!

**Comment by SimplyBallistic** [comment]
I believe that's it.

**Comment by SimplyBallistic** [comment]
That was painful. Didn't get to run it through an ide so there will most likely be a mistake or two

**Comment by SimplyBallistic** [comment]
Please bring back the Class ColorUtil. The removal of this class ruined a lot of code on this pull request.

**Comment by FlyingPikachu** [comment]
@SimplyBallistic The new changes look promising, I'll test it tomorrow (out of coffee).

Does your code depend on ColorUtil or is it a merge issue?

**Comment by SimplyBallistic** [comment]
Merge issue, I did a lot of code optimisation before you accepted the pull request by Greg. It happened right before I got the chance to pull hence the merge issue. I can try to fix it later without the need of that class. I am currently busy. @FlyingPikachu

File: src/main/java/com/evilmidget38/UUIDFetcher.java [hunk 1]
```diff
@@ -92,6 +92,6 @@ public static UUID fromBytes(byte[] array) {
     }
 
     public static UUID getUUIDOf(String name) throws Exception {
-        return new UUIDFetcher(Arrays.asList(name)).call().get(name);
+        return new UUIDFetcher(Collections.singletonList(name)).call().get(name);
```
Comment (FlyingPikachu) [inline src/main/java/com/evilmidget38/UUIDFetcher.java]: I'm thinking this is to save memory?

Comment (SimplyBallistic) [inline src/main/java/com/evilmidget38/UUIDFetcher.java]: Yep, instead of creating an entire list for no reason, the singleton list does this efficiently.

**Comment by FlyingPikachu** [comment]
@SimplyBallistic Using #215 I was able to recreate part of the bug using Quests 2.8.2-04 (does not include your fix), whereas a 'null' value was indeed placed in my player's data set. However, neither relogging nor restarting the server stopped the redo-delay from working. See my configuration [here](http://imgur.com/a/o2rEZ).

Shortly I'll take the same steps again, but with your changes. If a 'null' value is not created and everything works well as expected, I'll consider this fix satisfactory per our terms :)

**Comment by FlyingPikachu** [comment]
@SimplyBallistic Unfortunately, a 'null' value was still created upon completing the quest. I tried it twice over, to be sure. See [here](http://imgur.com/a/zlnSm).

**Comment by FlyingPikachu** [comment]
Resolved in #535

**Comment by SimplyBallistic** [comment]
Great, let me know how it goes and if there is anything out of place.

**Pull request closed without merging**